tests/test_bgp_af.rb

```diff
@@ -745,6 +544,7 @@ def default_metric(asn, vrf, af)
   def test_default_metric
     afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
     afs.each do |af|
+      config_ios_xr_dependencies('55') if platform == :ios_xr
```

Comment:
Passing the ASN as a string is very common in this file, not sure which is the preferred method?  Should it always be passed as a number instead?